openverse_catalog/dags/providers/provider_api_scripts/flickr.py

```diff
@@ -47,6 +50,14 @@ class FlickrDataIngester(ProviderDataIngester):
     batch_limit = 500
     retries = 5
 
+    max_records = 4_000
+    division_threshold = 20_000
+    min_divisions = 12
+    max_divisions = 60
+    # TODO expand on comment: We don't want to halt ingestion for Flickr when we
+    # get strange results with duplicates, just log it.
```

Comment:
Should there be more here?

Reply:
Oops, thought I caught all of those 😄 Done!